### PR TITLE
Pass Access and ID tokens to an upstream resource

### PR DESCRIPTION
**Pull request by pavel-mikhalchuk**
Pass Access and ID tokens to an upstream resource as headers "X-Access-Token" and "X-ID-Token" respectively.

### CODE REVIEW

File: kong/plugins/oidc/utils.lua [hunk 1]
```diff
@@ -67,6 +67,14 @@ function M.exit(httpStatusCode, message, ngxCode)
   ngx.exit(ngxCode)
 end
 
+function M.injectAccessToken(accessToken)
+  ngx.req.set_header("X-Access-Token", accessToken)
+end
+
+function M.injectIDToken(idToken)
+  ngx.req.set_header("X-ID-Token", idToken)
```
Comment (Trojan295) [inline kong/plugins/oidc/utils.lua]: The `idToken` is a Lua table, not a string. You need to handle it like the userinfo field (encode to a string and then to base64).
```suggestion
  local tokenStr = cjson.encode(idToken)
  ngx.req.set_header("X-ID-Token", ngx.encode_base64(tokenStr))
```

**Comment by pavel-mikhalchuk** [comment]
I've added couple more unit tests.

**Comment by pavel-mikhalchuk** [comment]
Also, Damian, please let me know if you'd like me to update readme file.

**Comment by Trojan295** [comment]
@pavel-mikhalchuk, the code looks ok, but I would be great to update also the README.md to show those new headers

**Comment by ronaldxl** [comment]
Hello,
Sorry for the intromission
Is this feature available in some way?
I need the acess token and id_token for make others requests

thanks!

**Comment by pavel-mikhalchuk** [comment]
@Trojan295, @ronaldxl,

Unfortunately I'm on the business trip right now and coming back December 10th. I'll try to work on this one this Saturday but can't guarantee that. We could merge this PR and update readme in different ticket so this feature is available.

**Pull request merged**